cukinia

```diff
@@ -196,6 +205,36 @@ _cukinia_http_request()
 	wget -q -O /dev/null "$url"
 }
 
+# _cukinia_mount: check for a specific mount point
+# arg1: the device
+# arg2: the mountpoint
+# arg@: (optinal) a list of options to look for
+_cukinia_mount()
+{
+	local device="$1"
+	local mountpoint="$2"
+	shift 2
+	local options="$@"
+	local found=0
+	local result
+
+	_cukinia_prepare "Checking mount: $device on $mountpoint with ${options:-any} options"
+
+	result=$(awk ' $1=="'$device'" && $2=="'$mountpoint'" { print $4 }' /proc/mounts | uniq)
+
+	[ -n "$result" ] || return 1
+
+	# ensure all mount options are set
+	for term in $options; do
+		if echo $result | grep -qw "$term"; then
+			found=$((found+1))
```

Comment:
there's other arithmetic in cukinia and they use a space before and after + so I'd say keep the style consistent